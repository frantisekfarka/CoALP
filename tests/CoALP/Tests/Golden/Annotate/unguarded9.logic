q(G,s(X)):-p(G,s(X)).
p(s(X),G):-q(s(X),G).
