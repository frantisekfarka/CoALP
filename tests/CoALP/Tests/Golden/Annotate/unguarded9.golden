q(V_2, s(V_3), transform-func-1(V_6)) :- p(V_2, s(V_3), V_6).
p(s(V_4), V_5, transform-func-2(V_-7)) :- q(s(V_4), V_5, V_-7).
